driver.sh

```diff
@@ -129,6 +129,14 @@ build_linux() {
   # is good about figuring out what needs to be rebuilt
   [[ -n "${cleanup:-}" ]] && mako_reactor mrproper
   mako_reactor ${config}
+  # If we're using a defconfig, enable some more common config options
+  # like debugging, selftests, and common drivers
+  if [[ ${config} =~ defconfig ]]; then
+    cat ../configs/common.config >> .config
+    # Some torture test configs cause issues on x86_64
+    [[ $ARCH != "x86_64" ]] && cat ../common/tt.config >> .config
+  fi
+  mako_reactor olddefconfig
```

Comment:
neat, I did not know about `olddefconfig`.

Reply:
does it always print these warnings?
```
.config:4409:warning: override: reassigning to symbol EXPERT
```

Reply:
I believe so given that the options are being overridden. I can hide that behind a `>/dev/null` if necessary.